specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2022-10-31-preview/iotcentral.json

```diff
@@ -0,0 +1,9369 @@
+          },
+          "Create or update a dashboard with LKV tile": {
+            "$ref": "./examples/dashboards_create_lkv.json"
+          },
+          "Create or update a dashboard with bar chart tile": {
+            "$ref": "./examples/dashboards_create_bar.json"
+          },
+          "Create or update a dashboard with count tile": {
+            "$ref": "./examples/dashboards_create_count.json"
+          },
+          "Create or update a dashboard with event history tile": {
+            "$ref": "./examples/dashboards_create_event_history.json"
+          },
+          "Create or update a dashboard with event tile": {
+            "$ref": "./examples/dashboards_create_event.json"
+          },
+          "Create or update a dashboard with heatmap chart tile": {
+            "$ref": "./examples/dashboards_create_heatmap.json"
+          },
+          "Create or update a dashboard with line chart tile": {
+            "$ref": "./examples/dashboards_create_line.json"
+          },
+          "Create or update a dashboard with map tile": {
+            "$ref": "./examples/dashboards_create_map.json"
+          },
+          "Create or update a dashboard with pie chart tile": {
+            "$ref": "./examples/dashboards_create_pie.json"
+          },
+          "Create or update a dashboard with state history tile": {
+            "$ref": "./examples/dashboards_create_state_history.json"
+          },
+          "Create or update a dashboard with state tile": {
+            "$ref": "./examples/dashboards_create_state.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DashboardId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Dashboard definition.",
+            "schema": {
+              "$ref": "#/definitions/Dashboard"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Dashboard"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Dashboards_Update",
+        "summary": "Update a dashboard",
+        "x-ms-examples": {
+          "Update a dashboard": {
+            "$ref": "./examples/dashboards_update.json"
+          }
+        },
+        "consumes": [
+          "application/json",
```

Comment:
When we do a `PATCH`, this should be ONLY `application/merge-patch+json`, not both.

Reply:
Fixed.